tests/test_events.py

```diff
@@ -1,6 +1,7 @@
+from requests_mock import Adapter
```

Comment:
Did `isort` make you change these? Looks like they may have released a new version that got smarter about what's a local package and what isn't...

Reply:
Yeah, the `tox -e check` command flagged all of these using the latest versions.
Sorting via either VSCode commands or `isort` will sort the imports correctly.